dev-lang/idris/idris-1.3.3-r1.ebuild

```diff
@@ -70,6 +70,9 @@ src_prepare() {
 	# runs dist/build/idris directly and breaks sandboxing
 	export LD_LIBRARY_PATH="${S}/dist/build${LD_LIBRARY_PATH+:}${LD_LIBRARY_PATH}"
 
+	# Upstream did this in commit cae9aa29
+	cabal_chdeps 'Cabal >= 2.4 && < 3.1' 'Cabal >= 2.4'
```

Comment:
I believe that this may already be in the ebuild since a while. Double check line 81 a bit further down from here. I didn't add it in, but I found it already present when adding in a `cabal_chdep` line for `haskeline-0.8`.

Reply:
Yes it is already in there. I don't know how I missed this, I must have been tracking down some other build error...